models/inline_strconv_parse.go

```diff
@@ -30,15 +29,6 @@ func parseBoolBytes(b []byte) (bool, error) {
 }
 
 // unsafeBytesToString converts a []byte to a string without a heap allocation.
-//
-// It is unsafe, and is intended to prepare input to short-lived functions
-// that require strings.
 func unsafeBytesToString(in []byte) string {
-	src := *(*reflect.SliceHeader)(unsafe.Pointer(&in))
-	dst := reflect.StringHeader{
-		Data: src.Data,
-		Len:  src.Len,
-	}
-	s := *(*string)(unsafe.Pointer(&dst))
-	return s
+	return *(*string)(unsafe.Pointer(&in))
```

Comment:
`go vet` complained about this.

Use the same code as 2.x for this: https://github.com/influxdata/influxdb/blob/master/models/inline_strconv_parse.go#L31-L34

Reply:
The trick is that (currently) the internal representation of string is the same as slice, except slice has an extra 'capacity'. So if you bitwise-interpret a byte slice as a string, it 'just works'.